include/eve/module/real/core/function/regular/simd/x86/ceil.hpp

```diff
@@ -20,21 +20,20 @@ namespace eve::detail
   //-----------------------------------------------------------------------------------------------
   // 128 bits implementation
   template<floating_real_scalar_value T, typename N, typename ABI>
-  EVE_FORCEINLINE wide<T, N, x86_128_> ceil_(EVE_SUPPORTS(sse4_1_)
-                                        , wide<T, N, x86_128_> const &a0) noexcept
```

Comment:
will we change those to if constexpr later ?

Reply:
I just didn't find the funcitons. Will fix.

Reply:
Done